.github/workflows/test-frontend.yml

```diff
@@ -4,7 +4,7 @@ on: [push]
 jobs:
   cypress-run:
     if: github.repository_owner == 'compiler-explorer'
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
best this stays 20.04 to remain compatible with our production environment and get more reliable tests